packages/ember-testing/lib/helpers.js

```diff
@@ -16,21 +17,26 @@ import RSVP from 'ember-runtime/ext/rsvp';
 var helper = Test.registerHelper;
 var asyncHelper = Test.registerAsyncHelper;
 
-function currentRouteName(app) {
-  var appController = app.__container__.lookup('controller:application');
+function lookup(app, path) {
+  if (app instanceof ApplicationInstance) {
+    return app.lookup(path);
+  } else {
+    return app.__container__.lookup(path);
+  }
+}
 
+function currentRouteName(app) {
+  var appController = lookup(app, 'controller:application');
```

Comment:
Unrelated to this PR directly, but we should update these to use `service:-routing` instead of `controller:application`.